Makefile

```
@@ -22,4 +22,5 @@ doxygen:
 	doxygen Doxyfile
 format: #TODO
 
-tidy: # TODO
\ No newline at end of file
+tidy: # TODO
+
```

вместо хранения в системе контроля версиями файла `.clang-format` добавить:  ?!

```suggestion
format:
	wget -nc https://raw.githubusercontent.com/torvalds/linux/master/.clang-format
	/usr/local/bin/clang-format -i ./src/* ./test/*
```

---

Нит! Ответил выше :)

---

И clang-format вызывать не надо. Он по разному может называться. У меня, например, -clang-format-11

---

ну .... всмысле не надо? а как же добиться форматирования? к тому же у меня он тоже по другому называется... это мягкая ссылка (еще один пункт, который им было бы не плохо освоить)

---

форматирование входит в процесс рефакторинга, а это часть разработки. Вот и пусть работают.